product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Known issues and additional notes"
+
+---
+
+This page lists known issues and additional notes for:
```

Comment:
Change "known issues and additional notes" to "known issues, limitations, and notes".

Reply:
Done.